src/Registry.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+/**
+@title Simple Name Registry
+@author hashedMae
+@notice A simple name registry that stores the data on chain
+ */
+contract Registry {
+
+
+    ///@notice Mapping of a name to address of currentOwner
+     mapping(string => address) public nameToUser;
+
+    ///@dev events for tracking when names are registered or released
+    event NameRegistered(string name, address user);
+    event NameReleased(string name, address user);
```

Comment:
This contract only deals with names, so we can shorten the events to `Registered` and `Released`.

Also, in functions and events, I like to follow an order for the parameters according to usage. Contract addresses first, then user addresses, then common parameters, then flags and secondary parameters.

In this case, I would probably do `event Released(address user, string name);`